src/bitmap/store.rs

```diff
@@ -189,8 +217,8 @@ impl Store {
                     *word = 0;
                 }
                 // end key bits
-                removed += (bits[end_key] & (!0u64).wrapping_shr(64 - end_bit)).count_ones();
-                bits[end_key] &= !(!0u64).wrapping_shr(64 - end_bit);
+                removed += (bits[end_key] & (!0u64 >> (63 - end_bit))).count_ones();
+                bits[end_key] &= !(!0u64 >> (63 - end_bit));
```

Comment:
Same here, why do you remove the `wrapping_shr`?

Reply:
Let's talk about the `wrapping_shr` in the upper conversation and I would mark the below resolved.